script.js

```diff
@@ -24,4 +25,106 @@ $(document).ready(function() {
 function handleResponse(response) {
   console.log(response);
   // add stuff here!
-}
\ No newline at end of file
+  for (var i=0; i < response.data.length; i++)
+  {
+    var image_link = response.data[i].images.standard_resolution.url;
+    // var img = $("<img />").attr('src', image_link)
+    var img = $("<img src='" + image_link + "' />");
+    $("#list").append(img);
+    $("#list").append(response.data[i].caption.text); 
```

Comment:
Helps to add a <div> wrapper around img+caption since it's like a single data-set.
Also `response.data[i].caption.text` will return null if there is no caption and it won't look purdy.